src/current/cockroachcloud/architecture.md

```diff
@@ -78,18 +78,22 @@ All CockroachDB {{ site.data.products.cloud }} organizations are given 50 millio
 
 #### Paid
 
-You must enter billing information and set [resource limits](#resource-limits) if you've already created one free CockroachDB {{ site.data.products.serverless }} cluster. Higher resource limits will allow your cluster to scale to meet your application's needs and maintain a high level of performance. You can set your storage and RU limits separately to reflect your usage, or choose an unlimited amount of resources to prevent your cluster from ever being throttled or disabled.
+You must enter billing information and set [resource limits](#resource-limits) if you've already created one free CockroachDB {{ site.data.products.basic }} cluster. Higher resource limits will allow your cluster to scale to meet your application's needs and maintain a high level of performance. You can set your storage and RU limits separately to reflect your usage, or choose an unlimited amount of resources to prevent your cluster from ever being throttled or disabled.
```

Comment:
This makes it sound like customers *must* resource limits, even though it's optional for basic clusters. For standard clusters, they do need to set the provisioned capacity limit, however.

Reply:
Hmmm I don't think this section needs to talk about resource limits at all (since we talk about them in the Performance section). Rewriting this now.